hazelcast/src/main/java/com/hazelcast/internal/config/MemberDomConfigProcessor.java

```diff
@@ -738,7 +738,7 @@ protected void handleSplitBrainProtectionNode(Node node, SplitBrainProtectionCon
                     !isNullOrEmpty(splitBrainProtectionConfig.getFunctionClassName());
             if (splitBrainProtectionFunctionDefinedByClassName) {
                 throw new InvalidConfigurationException("A split brain protection cannot simultaneously"
-                        + " define probabilistic-split-brain-protectionm or "
+                        + " define probabilistic-split-brain-protection or "
```

Comment:
I think this is a user facing change -- afaict the old message is wrong and misleading.